simple.css

```diff
@@ -448,32 +452,28 @@ input[type="radio"]:checked {
   background-color: var(--accent);
 }
 
+
```

Comment:
This extra blank like appears to have been added my mistake.

Reply:
I don't see this in upstream after cherry-picking the commits, resolving.